libp2p/peerinfo.nim

```diff
@@ -52,14 +76,24 @@ proc new*(
       key.getPublicKey().tryGet()
     except CatchableError:
       raise newException(PeerInfoError, "invalid private key")
+  
+  let peerId = PeerID.init(key).tryGet()
+
+  # TODO: should using signed peer records be configurable?
```

Comment:
Related to https://github.com/status-im/nim-libp2p/issues/429
We can handle this as part of 429, leave as is for now